udp-client/src/main/java/com/statful/client/core/udp/StatfulFactory.java

```diff
@@ -11,17 +12,17 @@
  */
 public final class StatfulFactory {
 
-    private static final UDPClientFactory UDP_CLIENT_FACTORY = new UDPClientFactory();
+    private static final UDPClientFactory udpClientFactory = new UDPClientFactory();
```

Comment:
Can you please revert the naming convention change? In the end this is a constant.